composer.json

```diff
@@ -254,7 +254,7 @@
         "drupal/custom_formatters": "3.0-alpha1",
         "drupal/token_filter": "1.0-beta1",
         "drupal/views_field_formatter": "1.5",
-        "drupal/lndr": "1.11",
+        "drupal/lndr": "1.*",
```

Comment:
in some cases this will install the first stable version in 1.x branch of lndr, which mase no sense
let's use ```~1.11``` to get anything equal or bigger